refs/biblio.json

```diff
@@ -2425,6 +2425,13 @@
     "SMPTE2052-1": {
         "aliasOf": "SMPTE2052-1-2013"
     },
+    "SMPTE296": {
+        "title": "ST 296:2012 - SMPTE Standard - 1280 × 720 Progressive Image 4:2:2 and 4:4:4 Sample Structure — Analog and Digital Representation and Analog Interface",
+        "href": "https://ieeexplore.ieee.org/document/7291722",
```

Comment:
The permalink is `https://doi.org/10.5594/SMPTE.ST296.2012`

Reply:
Aha, thanks.

Reply:
done